Subject: silence warning in fdisk_osf
From: Robert Schwebel <r.schwebel@pengutronix.de>

Silence this warning, which crashes when CONFIG_WERROR is active:

cc1: warnings being treated as errors
In file included from util-linux/fdisk.c:622:
util-linux/fdisk_osf.c: In function 'xbsd_writelabel':
util-linux/fdisk_osf.c:961: error: unused parameter 'p'

Signed-off-by: Robert Schwebel <r.schwebel@pengutronix.de>

used cleaner fix:

Signed-off-by: Marc Kleine-Budde <mkl@pengutronix.de>

---
 util-linux/fdisk_osf.c |    1 +
 1 file changed, 1 insertion(+)

Index: busybox-1.13.2/util-linux/fdisk_osf.c
===================================================================
--- busybox-1.13.2.orig/util-linux/fdisk_osf.c
+++ busybox-1.13.2/util-linux/fdisk_osf.c
@@ -966,6 +966,7 @@ xbsd_writelabel(struct partition *p)
 #if !defined(__alpha__) && !defined(__powerpc__) && !defined(__hppa__)
 	sector = get_start_sect(p) + BSD_LABELSECTOR;
 #else
+	(void)p;	/* silence warning */
 	sector = BSD_LABELSECTOR;
 #endif
 
